pkgs/by-name/do/docker-color-output/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "docker-color-output";
+  version = "2.5.1";
+
+  src = fetchFromGitHub {
+    owner = "devemio";
+    repo = pname;
```

```suggestion
    repo = "docker-color-output";
```

https://github.com/NixOS/nixpkgs/issues/277994